Doc/bytes_mode.rst

```diff
@@ -48,14 +48,15 @@ In Python 3, text values are represented as ``str``, the Unicode text type.
 In Python 2, the behavior of python-ldap 3.0 is influenced by a ``bytes_mode``
 argument to :func:`ldap.initialize`:
 
-``bytes_mode=True`` (backwards compatible):
+``bytes_mode=ldap.BYTES_MODE_LEGACY`` (backwards compatible):
     Text values are represented as bytes (``str``) encoded using UTF-8.
 
-``bytes_mode=False`` (future compatible):
+``bytes_mode=ldap.BYTES_MODE_MODERN`` (future compatible):
     Text values are represented as ``unicode``.
 
-If not given explicitly, python-ldap will default to ``bytes_mode=True``,
-but if an ``unicode`` value supplied to it, it will warn and use that value.
```

Comment:
but if an ``unicode`` value *is* supplied to it, it will warn and use that value.